autoload/nnn.vim

```diff
@@ -2,6 +2,14 @@ let s:temp_file = ""
 let s:action = ""
 let s:tbuf = 0
 
+" HACK: cannot use / in a session name
+let s:local_ses = 'nnn_vim_'.substitute(tempname(), '/', '_', 'g')
```

Comment:
Maybe we don't need tempname anymore since we append it with datetime. Also to keep it short.

Reply:
Got it. How about only using tempname if strftime is not available?

Reply:
That's what I ended up doing, seems the most reasonable.